packages/bug-detectors/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"extends": "../../tsconfig.json",
+	"compilerOptions": {
+		"rootDir": ".",
+		"outDir": "dist"
+	},
+	"references": [
+		{
+			"path": "../instrumentor"
+		},
+		{
+			"path": "../hooking"
```

Comment:
Missing `fuzzer` dependency?

Reply:
I wonder, why did the compilation worked at all?